content/source/docs/cloud/api/policies.html.md

```diff
@@ -22,6 +22,8 @@ page_title: "Policies - API Docs - Terraform Cloud"
 
 # Policies API
 
+-> **Note:** Sentinel is a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
```

Comment:
Not sure if we want to say straight up "Sentinel" itself is a paid feature (since the OSS binary is free) vs saying "Sentintel Policy framework for Terraform Cloud" or something like that. Probably worth looping @glenngillen into this

Reply:
Hrm, I'm torn. I think we could do better but from a quick search this is at least consistent with the way it's stated in other places. If we are going to change it then I think in this context I'd drop the reference to Sentinel entirely. You're not directly interfacing with Sentinel via this API, you're providing the things it will later use. 

So maybe something more generic like "Policies are a..." or "Automated Goverance via Policy as Code..." or ... ?

Reply:
Yeah, so part of what I'm going for with these is to be careful enough with the reader's attention/patience/pattern-recognition that one can understand the whole note without actually "reading" it per se. 

So while Sentinel's the framework, not the feature, it's used so heavily in our marketing and in community conversations that it would be confusing _not_ to use that name here. 

How about "Sentinel policies are a paid feature"? I think that's still glance-able enough to be kind to our readers, while also helping differentiate between the feature and the framework a bit.